laravel/resources/views/pages/slot/view.blade.php

```diff
@@ -154,5 +157,13 @@
         @endif
 
         <a href="/event/{{ $slot->event->id }}" class="btn btn-primary">Back to Event</a>
+
+        @if(Auth::user()->hasRole('admin')&& $taken==true)
+        <p>
+            Are you sure you want to remove this user for this shift?
```

Comment:
More unrelated code...

Please make sure your pull requests only include the changes you are describing